### PR TITLE
add new files

### PR DESCRIPTION
**Pull request by shreyas-9601**

### CODE REVIEW

File: Database.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from mongoengine import *
+
+connect("ordersdb")
+
+class OrderMongodb(Document):   
+    order_id = StringField(required=True)
```
Comment (bikashgupta-zinrelo) [inline Database.py]: Please make order id unique

File: Database.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    name = StringField(required=True)
+    birthday = StringField(required=True)
```
Comment (bikashgupta-zinrelo) [inline Database.py]: Please change the type to Datetimefield

File: Database.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from mongoengine import *
```
Comment (bikashgupta-zinrelo) [inline Database.py]: Please change the file name to orders_model.py

File: Orders_model.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+from mongoengine import *
```
Comment (shreyas-9601) [inline Orders_model.py]: rename file orders_model

File: Process_orders_data.py [hunk 5]
```diff
@@ -73,18 +74,23 @@ def process(self):
...
                 email = row[email_index]
                 state = row[state_index]
                 zipcode = row[zipcode_index]
                 
                 order=Order(order_id, name, birthday, email, state, zipcode)
+                
+                ordermodel=OrdersModel(order_id=order_id, name=name, birthday=birthday, email=email, state=state, zipcode=zipcode)
+                OrdersModel.save(ordermodel)
 
                 if (order.validate_orders()):
```
Comment (shreyas-9601) [inline Process_orders_data.py]: remove this code

File: Process_orders_data.py [hunk 6]
```diff
@@ -73,18 +74,23 @@ def process(self):
...
                 name = row[name_index]
-                birthday = row[birthday_index]
+                birthday=datetime.strptime(row[birthday_index], '%m/%d/%Y')
                 email = row[email_index]
                 state = row[state_index]
                 zipcode = row[zipcode_index]
                 
                 order=Order(order_id, name, birthday, email, state, zipcode)
+                
+                ordermodel=OrdersModel(order_id=order_id, name=name, birthday=birthday, email=email, state=state, zipcode=zipcode)
```
Comment (shreyas-9601) [inline Process_orders_data.py]: read save order method in order class and use it

**Review by bikashgupta-zinrelo — APPROVED** [review]

**Pull request merged**